rbi/stdlib/reline.rbi

```diff
@@ -0,0 +1,223 @@
+    attr_reder :special_prefixes
+    attr_reder :completion_proc
+    attr_reder :output_modifier_proc
+    attr_reder :prompt_proc
+    attr_reder :auto_indent_proc
+    attr_reder :pre_input_hook
+    attr_reder :dig_perfect_match_proc
+
+    attr_accessor :config
+    attr_accessor :key_stroke
+    attr_accessor :line_editor
+    attr_accessor :last_incremental_search
+    attr_reader :output
+
+    extend Forwardable
+    def_delegators :config,
+      :autocompletion,
+      :autocompletion=,
+      *DIALOG_COLOR_APIS
+
+    def encoding; end
+
+    def completion_append_character=(val); end
+
+    def basic_word_break_characters=(v); end
+
+    def completer_word_break_characters=(v); end
+
+    def basic_quote_characters=(v); end
+
+    def completer_quote_characters=(v); end
+
+    def filename_quote_characters=(v); end
+
+    def special_prefixes=(v); end
+
+    def completion_case_fold=(v); end
+
+    def completion_case_fold; end
+
+    def completion_quote_character; end
+
+    def completion_proc=(p); end
+
+    def output_modifier_proc=(p); end
+
+    def prompt_proc=(p); end
+
+    def auto_indent_proc=(p);  end
+
+    def pre_input_hook=(p); end
+
+    def dig_perfect_match_proc=(p); end
+
+    DialogProc = Struct.new(:dialog_proc, :context)
+
+    def add_dialog_proc(name_sym, p, context = nil); end
+
+    def dialog_proc(name_sym); end
+
+    def input=(val); end
+
+    def output=(val); end
+
+    def vi_editing_mode; end
+
+    def emacs_editing_mode; end
+
+    def vi_editing_mode?; end
+
+    def emacs_editing_mode?; end
+
+    def get_screen_size; end
+
+    Reline::DEFAULT_DIALOG_PROC_AUTOCOMPLETE = ->() { }
+    Reline::DEFAULT_DIALOG_CONTEXT = Array.new
+
+    def readmultiline(prompt = '', add_hist = false, &confirm_multiline_termination); end
+
+    def readline(prompt = '', add_hist = false); end
+
+    def ambiguous_width; end
+  end
+
+  extend Forwardable
+  extend SingleForwardable
+
+  def_single_delegators :core, :completion_append_character, :completion_append_character=
+  def_single_delegators :core, :basic_word_break_characters, :basic_word_break_characters=
+  def_single_delegators :core, :completer_word_break_characters, :completer_word_break_characters=
+  def_single_delegators :core, :basic_quote_characters, :basic_quote_characters=
+  def_single_delegators :core, :completer_quote_characters, :completer_quote_characters=
+  def_single_delegators :core, :filename_quote_characters, :filename_quote_characters=
+  def_single_delegators :core, :special_prefixes, :special_prefixes=
+  def_single_delegators :core, :completion_proc, :completion_proc=
+  def_single_delegators :core, :output_modifier_proc, :output_modifier_proc=
+  def_single_delegators :core, :prompt_proc, :prompt_proc=
+  def_single_delegators :core, :auto_indent_proc, :auto_indent_proc=
+  def_single_delegators :core, :pre_input_hook, :pre_input_hook=
+  def_single_delegators :core, :dig_perfect_match_proc, :dig_perfect_match_proc=
```

Comment:
Sorbet does not understand these methods, and thus these methods will not be present.

I'd prefer we delete these, or declare what methods will actually be defined using normal `def` nodes.